src/SFA.DAS.FAA.Web.UnitTests/Controllers/Apply/WhenPostingApplicationSubmitted.cs

```diff
@@ -26,13 +48,69 @@ public async Task And_User_Selected_Yes_Then_Redirect_To_Equality_Flow(
     [Test, MoqAutoData]
     public async Task And_User_Selected_No_Then_Redirect_To_Their_Applications(
         ApplicationSubmittedViewModel model,
+        string bannerMessage,
+        string govIdentifier,
+        Guid candidateId,
+        Guid applicationId,
+        GetApplicationSubmittedQueryResponse response,
+        [Frozen] Mock<IMediator> mediator,
+        [Frozen] Mock<ICacheStorageService> cacheStorageService,
         [Greedy] ApplyController controller)
     {
+        mediator.Setup(x => x.Send(It.Is<GetApplicationSubmittedQuery>(c =>
+                c.CandidateId.Equals(candidateId)
+                && c.ApplicationId.Equals(applicationId)), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(response);
+        var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+        {
+            new(CustomClaims.CandidateId, candidateId.ToString()),
+        }));
+        controller.ControllerContext = new ControllerContext
+        {
+            HttpContext = new DefaultHttpContext { User = user }
+        };
         model.AnswerEqualityQuestions = false;
+        cacheStorageService.Setup(x => x.Get<string>($"{govIdentifier}-ApplicationSubmitted"))
+            .ReturnsAsync(bannerMessage);
 
         var result = await controller.ApplicationSubmitted(model) as RedirectToRouteResult;
 
         result.Should().NotBeNull();
-        result!.RouteName.Should().Be(RouteNames.ApplyApprenticeship.ApplicationSubmittedConfirmation);
+        result!.RouteName.Should().Be(RouteNames.Applications.ViewApplications);
```

Comment:
I think here as well we can add this or similar:
```
cacheStorageService.Verify(x=>x.Set($"{govUkIdentifier}-ApplicationSubmitted", It.IsAny<object>(),1,1), Times.Once);
```

Reply:
Added additional assert to verify the call